source/style-guide/seo-guidelines.txt

```diff
@@ -9,6 +9,45 @@ enter into search engines (keywords) and employs best practices to
 improve traffic to web pages from search engines. The following 
 guidance should help you optimize your documentation pages for search.
 
+.. important::
+
+   Our taxonomy efforts also improve findability and increase 
+   relevance for search queries. Our taxonomy comes with its own set of 
+   instructions and best practices. To learn more, see 
+   `Taxonomy tagging instructions 
+   <https://wiki.corp.mongodb.com/display/DE/Taxonomy+tagging+instructions>`__. 
+
+Keywords
+--------
+
+You can add the actual terms that users enter into search engines 
+(keywords) to the body of your page to improve its SEO.
+
+.. important::
+  
+   This guidance makes a distinction between the terms "keywords" and 
+   "keyword meta tags". While keyword meta tags are available, some 
+   search engines might ignore these tags. To learn more, see 
```

Comment:
```suggestion
   search engines might ignore these tags. To learn more about tags, see 
```
Initially thought this link would lead to further explanation of difference between 'keywords' and 'keyword meta tags'